docs/millennium-os/manual/chapters/40_gcodes.md

```diff
@@ -115,7 +115,7 @@ Looks up stored rotation values from the given workplace number (or the current
 M6515 [Xnnn] [Ynnn] [Znnn]
 ```
 
-Takes at least one of X, Y and Z co-ordinates and checks that they are within the axes limits of the machine, otherwise triggers an abort. This is used by other macros to make sure we do not try to move outside of machine limits.
+Provide at least one of X, Y, and Z coordinates to check that they are within the axes limits of the machine, and trigger an abort if they are outside the work envelope. This is used by other macros to make sure we do not try to move outside of machine limits.
```

Comment:
I think maybe 'Accepts' would work better here. 'Provide' suggests we're talking from the perspective of the operator but then 'and trigger' is not something the operator is doing. The perspective should be consistent for both, or specified where unclear.

```suggestion
Accepts at least one of X, Y, and Z coordinates to check that they are within the axes limits of the machine, and trigger an abort if they are outside the work envelope. This is used by other macros to make sure we do not try to move outside of machine limits.
```